index.html

```diff
@@ -0,0 +1,139 @@
+  <meta charset="UTF-8">
+  <title>Document</title>
+  <link rel="stylesheet" href="./styles/styles.css">
+</head>
+
+<body>
+  <main id="main_viewport">
+
+    <div class="Nav-Bar">
+      <div class="box box1">
+        <a href="#">
+          <img class="img_btn" src="./img/btn_normal.png" alt="">
+          <img class="btn-icon" src="./img/menuicons.png" alt="">
+        </a>
+      </div>
+      <div class="box box2">
+        <a href="#">
+          <img class="img_btn" src="./img/btn_normal.png" alt="">
+          <img class="btn-icon" src="./img/menuicons.png" alt="">
+        </a>
+      </div>
+      <div class="box box3">
+        <a href="#">
+          <img class="img_btn" src="./img/btn_normal.png" alt="">
+          <img class="btn-icon" src="./img/menuicons.png" alt="">
+        </a>
+      </div>
+      <div class="box box4">
+        <a href="#">
+          <img class="img_btn" src="./img/btn_normal.png" alt="">
+          <img class="btn-icon" src="./img/menuicons.png" alt="">
+        </a>
+      </div>
+      <div class="box box5">
+        <a href="#">
+          <img class="img_btn" src="./img/btn_normal.png" alt="">
+          <img class="btn-icon" src="./img/menuicons.png" alt="">
+        </a>
+      </div>
+      <div class="box box6">
+        <a href="#">
+          <img class="img_btn" src="./img/btn_normal.png" alt="">
+          <img class="btn-icon" src="./img/menuicons.png" alt="">
+        </a>
+      </div>
+      <div class="box box7">
+        <a href="#">
+          <img class="img_btn" src="./img/btn_normal.png" alt="">
+          <img class="btn-icon" src="./img/menuicons.png" alt="">
+        </a>
+      </div>
+      <div class="box box8">
+        <a href="#">
+          <img class="img_btn" src="./img/btn_normal.png" alt="">
+          <img class="btn-icon" src="./img/menuicons.png" alt="">
+        </a>
+      </div>
+      <div class="box box9" style="min-height: 16px;">
+          <img class="globalsearch" src="./img/globalsearch_field.png" alt="">
+      </div>
+    </div>
+    <header class="main-header">
+      <section class "main-text">
+        <img id="display-text1" src="./img/your-verse-small.png" alt="what is your verse">
+      </section>
+      <section class="sub-header">
+        <img src="./img/everyone-share.png" alt="Everyone Share">
+      </section>
+      <div class="watch-button">
+        <a class="watch-anchor" href="#">
+          <img class="watch-img" src="./img/watch-film-btn.png" alt="Watch the film">
+        </a>
+      </div>
+    </header>
+    <section class="promos">
+      <div class="promo promo1">
+        <img src="./img/promo1.jpg" alt="">
+      </div>
+      <div class="promo promo2">
+        <img src="./img/promo2.jpg" alt="">
+      </div>
+      <div class="promo promo3">
+        <img src="./img/happy_birthday.jpg" alt="">
+      </div>
+      <div class="promo promo4">
+        <img src="./img/supplier_resp.jpg" alt="">
+      </div>
+    </section>
```

Comment:
:no_good_woman:  remove all **\<img\>** elements in the **promo section** as they have to be background images

Reply:
done